kubeadm/scripts/PrepareNode.ps1

```diff
@@ -67,24 +54,9 @@ $env:Path += ";$global:KubernetesPath"
 
 DownloadFile $kubeletBinPath https://dl.k8s.io/$KubernetesVersion/bin/windows/amd64/kubelet.exe
 DownloadFile "$global:KubernetesPath\kubeadm.exe" https://dl.k8s.io/$KubernetesVersion/bin/windows/amd64/kubeadm.exe
-DownloadFile "$global:KubernetesPath\wins.exe" https://github.com/rancher/wins/releases/download/v0.0.4/wins.exe
-
-if ($ContainerRuntime -eq "Docker") {
-    # Create host network to allow kubelet to schedule hostNetwork pods
-    # NOTE: For containerd the 0-containerd-nat.json network config template added by
-    # Install-containerd.ps1 joins pods to the host network.
-    Write-Host "Creating Docker host network"
-    docker network create -d nat host
-} elseif ($ContainerRuntime -eq "containerD") {
-    DownloadFile "c:\k\hns.psm1" https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1
-    Import-Module "c:\k\hns.psm1"
-    # TODO(marosset): check if network already exists before creatation
-    New-HnsNetwork -Type NAT -Name nat
-}
 
-Write-Host "Registering wins service"
-wins.exe srv app run --register
-start-service rancher-wins
+DownloadFile "c:\k\hns.psm1" https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1
+Import-Module "c:\k\hns.psm1"
```

Comment:
If this is needed by the CNI then it is included in the Host process CNI image, so not technically needed.